frappe/public/js/frappe/form/quick_entry.js

```diff
@@ -161,9 +161,13 @@ frappe.ui.form.QuickEntryForm = class QuickEntryForm {
 
 			if (data) {
 				me.dialog.working = true;
-				me.dialog.set_message(__("Saving..."));
 				me.insert().then(() => {
-					me.dialog.clear_message();
+					let messagetxt = __("Created new {0} '{1}'", [__(me.doctype), this.doc.name]);
+					me.dialog.animate = "slow";
+					me.dialog.hide();
+					setTimeout(function () {
+						frappe.show_alert({ message: messagetxt, indicator: "green" }, 3);
+					}, 1000);
```

Comment:
Not sure about this hardcoded timeout... Are there other ways to ensure things will happen in the right order?

Reply:
I’m not gonna say I didn’t try other solutions first.

Didn’t get the following to work though:
```
$("body").one('transitionend', '.modal-backdrop', function() {
	frappe.show_alert();
});
```

May try again if I find some time. But on the other hand, it seemed quite right to have the message overlap a bit with the transition, so we’d want the message to be triggered maybe some 0.2s (or 200ms) after the transition (which is 0.3s or 300ms long) starts, not ends.

Basically I think it’s much better than before, and about right. But of course there’s always a lot of room for further improvement…

Reply:
Cutting the hard-coded timeout in half as per @surajshetty3416 should reduce its impact. I fear that‘s about the best we can do at this point.